src/gs-discovery-feed-content-provider.c

```diff
@@ -182,7 +182,6 @@ search_done_cb (GObject *source,
 	 * internally randomized order. */
 	gs_app_list_filter (list, filter_for_discovery_feed_apps, NULL);
 	gs_app_list_randomize (list);
-	gs_app_list_sort (list, search_sort_by_kudo_cb, NULL);
```

Comment:
The function is unused now, no?

Reply:
It is! I must have forgotten to remove it.